efficientdet/keras/train.py

```diff
@@ -17,13 +17,12 @@
 from absl import app
 from absl import flags
 from absl import logging
+import tensorflow as tf
 
 import dataloader
 import hparams_config
 import utils
-import copy
-import tensorflow as tf
-from keras.train_lib import get_optimizer, get_callbacks, EfficientDetNetTrain, BoxLoss, BoxIouLoss, FocalLoss
```

Comment:
the formater doesn't allow multiple imports in a single line (typing is the only exception).